src/settings.h

```diff
@@ -10,6 +10,7 @@ enum alignment { left, center, right };
 enum ellipsize { start, middle, end };
 enum icon_position_t { icons_left, icons_right, icons_off };
 enum separator_color { FOREGROUND, AUTO, FRAME, CUSTOM };
+struct separator_color_data { enum separator_color type; char *sep_color; };
```

Comment:
This hurts readability, structs should always be defined the long way (one item per line).

Reply:
Full ACK. Done. I missed up the enum with a struct. 🙈